Userland/man.cpp

```diff
@@ -102,9 +99,9 @@ int main(int argc, char* argv[])
 
     dbg() << "Loading man page from " << file->filename();
     auto buffer = file->read_all();
-    String source { (const char*)buffer.data(), (size_t)buffer.size() };
+    String source{ (const char*)buffer.data(), (size_t)buffer.size() };
```

Comment:
:cry:

Reply:
@bugaevc Seems .clang-format said it should be so.

Reply:
@bugaevc tried multiple times.

Reply:
Please use `clang-format` version 8 or newer.

Reply:
looks like my clang-format is too old. Upgrade clang-format. It is OK now.